Website/Sources/Pages/HomeSections.swift

```diff
@@ -24,8 +25,8 @@ extension HomeSectionType {
 
   func isAvailable(for year: ConferenceYear) -> Bool {
     switch year {
-    case .year2025: true
-    case .year2026: [.about, .outline, .tickets, .access, .meetTheHosts, .sponsor, .meetTheOrganizers].contains(self)
+    case .year2025: [.about, .outline, .speaker, .timetable, .meetTheHosts, .sponsor, .meetTheOrganizers, .access].contains(self)
```

Comment:
Ticket and speaker recruitment items have been changed to display only the most recent pages.